capstone/src/main/webapp/style.css

```diff
@@ -108,3 +108,18 @@ ul {
 .clear {
   clear: both;
 }
+
+.button {
```

Comment:
This is good. Thanks for moving it to the style sheet, I understand that it's rather nitpicky but it enhances code clarity :)

Reply:
For future iterations on this button feature it may be important to change the class names of each button to make it easier to tell which style rules apply for which button

Reply:
This is great!